gcloud-java-contrib/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.gcloud</groupId>
+  <artifactId>gcloud-java-contrib</artifactId>
+  <packaging>jar</packaging>
+  <name>GCloud Java contrib</name>
```

Comment:
Nit: could write out "contributions" here just to be consistent with other modules' pom.xmls